packages/rocketchat-mailer/server/api.js

```diff
@@ -0,0 +1,86 @@
+import s from 'underscore.string';
+import juice from 'juice';
+let contentHeader;
+let contentFooter;
+
+let body;
+let Settings = {
+	get: () => {},
+};
+
+export const replacekey = (str, key, value = '') => str.replace(new RegExp(`(\\[${ key }\\]|__${ key }__)`, 'igm'), value);
+
+export const translate = (str) => str.replace(/\{ ?([^\} ]+)(( ([^\}]+))+)? ?\}/gmi, (match, key) => TAPi18n.__(key));
+
+export const replace = function replace(str, data = {}) {
```

Comment:
What if we don't expose this function? I mean, instead of `send()` accepting an object with a param-replaced `html` property, we call it passing the template and the parameters to be replaced inside the module. This can avoid any accidental non-escaping.